src/main/webapp/app/shared/util/firebase/firebase-review-utils.ts

```diff
@@ -352,6 +355,9 @@ const findMutationEffectReviews = (
         { [fieldKey]: mutationEffect[reviewKey].lastReviewed },
         isNestedUnderCreateOrDelete(parentReview)
       );
+      if (fieldKey === 'oncogenic' || fieldKey === 'effect') {
```

Comment:
Solution works for now, but will break with new listener changes. Can leave as is, but might want to consider a fix that checks the input type or some way that will not break (if that is possible).

Reply:
Yeah this is true. Don't know if there's a good way to know if a field is a checkbox/radio without probably hard-coding. I will come back and fix it